src/assets/data/contributors.json

```
@@ -1,7 +1,7 @@
 [
   {
     "name": "Justin Berque",
-    "company": "Snapp'",
+    "company": "Webwag'",
```

maybe you want to also remove the `'`?
```suggestion
    "company": "Webwag",
```